parser/src/ast/evaluator_function.rs

```diff
@@ -0,0 +1,73 @@
+use super::{Identifier, IntegrityStmt, TraceCols};
+
+/// Evaluator functions take description of the main and auxiliary traces as input, and returns
+/// integrity constraints that are enforced on those trace columns.
```

Comment:
As noted in #122 and #213, we may need to allow passing periodic columns and random values.

Also, I would rephrase the end of this from "and returns..." to "and enforces integrity constraints on those trace columns."